### PR TITLE
Starter Kits

### LINKED ISSUE #45: Guides - Starter Kits
A curated list of starter kits that get you started for a Webpack-based project:
- [Angular Full-Stack Generator](https://github.com/angular-fullstack/generator-angular-fullstack)
- [Angular2 Webpack Starter](https://github.com/AngularClass/angular2-webpack-starter)
- [NG6 Starter](https://github.com/AngularClass/NG6-starter)
- [React Starter Kit](https://github.com/kriasoft/react-starter-kit)

### PR DESCRIPTION
**Pull request by skipjack**
Adds a starter kits guide fetched from @ahfarmer's [tool-list package](https://github.com/ahfarmer/tool-list). Just working on a better display for the tags column and then this should be good to go.

Resolves #45

### CODE REVIEW

File: components/starter-kits/starter-kits.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import Container from '../container/container';
+import Link from '../link/link';
+import Kits from './starter-kits-data.json';
+import './starter-kits-style';
+
+// NOTE: The table classes used in this component correspond to
+// those used in the markdown utility. Ideally we will soon create
+// a table component that both the markdown utility and this page
+// can use. This component could even use something like griddle
+// to allow sorting and such.
```
Comment (bebraw) [inline components/starter-kits/starter-kits.jsx]: If we want something super light, it's enough to attach a little standalone script (these exist for sure) that hooks into the table.

File: components/starter-kits/starter-kits.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                  <Link to={ kit.githubUrl }>{ kit.githubRepoName }</Link>
+                </div>
+              </div>
+              <div className="table-td">
+                <div className="table-td-title">Maintainer</div>
+                <div className="table-td-content">{ kit.githubUserName }</div>
+              </div>
+              <div className="table-td">
+                <div className="table-td-title">Tags</div>
+                <div className="table-td-content">{ kit.tags.join(' | ') }</div>
```
Comment (bebraw) [inline components/starter-kits/starter-kits.jsx]: It's ok. Maybe better as separate elements with some nice class, though?

**Comment by skipjack** [comment]
Mobile:

image

Normal:

image

**Comment by bebraw** [comment]
Looks good enough. 👍

File: components/starter-kits/starter-kits.jsx (hunk 1 above)
Comment (skipjack) [inline components/starter-kits/starter-kits.jsx]: I'd really like to hold off on anything table related until we get a better markdown solution in place. I think the best option by far is to pipe all markdown to components that contain clean, isolated logic. It just feels like we've added a lot of features at this point, many of them in a hacky way which have come back to bite us.

Comment (bebraw) [inline components/starter-kits/starter-kits.jsx]: Sure.

**Comment by skipjack** [comment]
Another one down, we just need to remember to add the link to this page in the sidebar. I'll add a todo for that in #1153.

**Pull request merged**